pkg

```diff
@@ -155,7 +155,7 @@ while getopts h78abdf:lopqvsSL opt; do
 	esac
 done
 
-PROJECTS="$("${COMPOSECMD[@]}" -f $COMPOSE_FILE config --services)"
+PROJECTS="$(cat infrastructure/docker/build/docker-compose.yml | grep -E '^\s\s[a-zA-Z_-]+:$' | sed 's/[: ]//g')"
```

Comment:
Parsing YAML with `grep` and `sed` does not seem so maintainable. What are your thoughts about rearranging the output of `docker-compose config --services` to have `weasel` first?

Reply:
That sounds like a better solution to me